modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/osgi/commands/PortalAddressOSGiCommands.java

```diff
@@ -51,11 +51,11 @@ public class PortalAddressOSGiCommands {
 	public void initializeCompanyCountries(long companyId) throws Exception {
 		_countryLocalService.deleteCompanyCountries(companyId);
 
-		populateCompanyCountries(companyId);
+		populateCompanyCountries(_companyLocalService.getCompany(companyId));
 	}
 
-	public void populateCompanyCountries(long companyId) throws Exception {
-		Company company = _companyLocalService.getCompany(companyId);
+	public void populateCompanyCountries(Company company) throws Exception {
+		long companyId = company.getCompanyId();
```

Comment:
Hey @MrJohn1911, just inline the `company.getCompanyId()` instead. No need to create this variable since there is no extra cost calling the `getCompanyId()` more times. Thx

Reply:
Also, you need to keep a method `populateCompanyCountries` that receives a `long companyId` as well so that this can be called in the gogo shell.